spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -295,6 +725,88 @@ spring:
 ----
 
 
+== Binder SPI
+
+== Miscellaneous
```

Comment:
move much of the new information that is relevant now into previous sections, e.g. partitioning.  Stuff that is from the previous release should likely be removed.  I realize this is a WIP PR, but this section should be cleaned up significantly if not entirely removed.